### PR TITLE
docs: Improve the basic setup guide

### PR DESCRIPTION
**Pull request by tara-det-ai**
Improve the basic setup guide with feedback from customer support.

### CODE REVIEW

File: docs/setup-cluster/basic-setup-guide/basic.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ Basic Setup
+#############
+
+If you have not used Determined before, and you want to quickly set up a new training environment,
+you are at the right place!
+
+.. note::
+
+   To set up a Determined training environment on-prem or on cloud, visit :ref:`Advanced Setup
+   <advanced-setup>`.
```
Comment (rb-determined-ai) [inline docs/setup-cluster/basic-setup-guide/basic.rst:15]: This needs re-phrasing; technically we _are_ about to set up a Determined training environment on-prem.

Maybe: 
```
To set up a Determined training environment on a shared server or cluster or in the cloud, visit :ref:`Advanced Setup <advanced-setup>`.
```

Honestly, the fact that it's called "Basic vs Advanced" is probably enough to motivate the fact that there are two docs.  This note could practically just say, "For advanced installations, see Advanced Setup".

File: docs/setup-cluster/basic-setup-guide/basic.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+**Prerequisites**
+
+-  :ref:`Installation Requirements <system-requirements>`
+
+*********************************
+ Step 1 - Install Docker Desktop
```
Comment (rb-determined-ai) [inline docs/setup-cluster/basic-setup-guide/basic.rst:23]: "Docker Desktop" is the mac-specific software.  This should probably just be called "Install Docker".

File: docs/setup-cluster/basic-setup-guide/basic.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+*********************************
+
+-  :ref:`Install Docker <install-docker>` on your machine.
```
Comment (rb-determined-ai) [inline docs/setup-cluster/basic-setup-guide/basic.rst:26]: linking to another doc is ok, but if there was a good time to use tabs, this is probably a good one.  It's nice if the basic setup guide is all-in-one.

File: docs/setup-cluster/basic-setup-guide/basic.rst (hunk 1 above)
Comment (tara-det-ai) [reply docs/setup-cluster/basic-setup-guide/basic.rst:15]: Edited.

For more advanced installations, visit :ref:`Advanced Setup <advanced-setup>`.

File: docs/setup-cluster/basic-setup-guide/basic.rst (hunk 3 above)
Comment (tara-det-ai) [reply docs/setup-cluster/basic-setup-guide/basic.rst:26]: I consider this resolved.

The "Install Docker" page itself is too complex to be a tab within this doc but I did simplify the Install Docker page by applying tabbed content to the first two steps.

File: docs/setup-cluster/basic-setup-guide/basic.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+*****************************
+ Step 2 - Install Determined
+*****************************
+
+Install the Determined library and start a cluster locally.
+
+-  Ensure you have Docker running and then run the following command:
```
[the remark this answers is not in the document]
Comment (tara-det-ai) [reply docs/setup-cluster/basic-setup-guide/basic.rst:34]: Resolved.

**Comment by tara-det-ai** [comment]
New direction: two PRs: basic setup and advanced setup options.

**Pull request closed without merging**